frontend/src/components/Checkbox/index.module.css

```diff
@@ -1,26 +1,30 @@
-.inputWrapper {
+.root {
   --size-checkbox: 24px;
 
   position: relative;
   display: flex;
   align-items: center;
 }
 
+.inputWrapper {
+  width: var(--size-checkbox);
+  margin-right: 20px;
+}
+
 .input {
+  position: absolute;
+  left: 0;
   width: var(--size-checkbox);
   height: var(--size-checkbox);
   opacity: 0;
 }
```

Comment:
Do we need any of this? Couldn't we achieve the same functionality and design with: 
```.css
.input {
  position: absolute;
  visibility: hidden;
}
```
?

Reply:
by defining visibility to be hidden the ☑️ doesn't show at all. 

What does work is:
```
.input {
  position: absolute;
  opacity: 0;
}
```